ci/scripts/pr_list_database.py

```diff
@@ -2,6 +2,7 @@
 
 import sys
 from pathlib import Path
+import argparse
```

Comment:
Aren't you already importing what you need from `argparse` on line 6?  What is this for?

Reply:
actually, I needed it for `argparse.REMAINDER`